### PR TITLE
Metrics docs

### PR DESCRIPTION
**Pull request by gregkalapos**
- Mainly stolen from Java (and of course adapted)
- Documents the `MetricsInterval` setting
- And documents all metrics that the agent currently sends

### CODE REVIEW

File: docs/metrics.asciidoc [hunk 1]
```diff
@@ -0,0 +1,102 @@
+ifdef::env-github[]
```
Comment (bmorelli25) [inline docs/metrics.asciidoc]: This file needs to be included in `index.asciidoc` for it to build:
```js
include::./metrics.asciidoc[Metrics]
```

Comment (gregkalapos) [reply docs/metrics.asciidoc]: Thanks, added - trying to have the same order as other agents, let me know if I should move it.

Comment (bmorelli25) [reply docs/metrics.asciidoc]: That's good for now. Thanks!

File: docs/metrics.asciidoc [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
```
Comment (SergeyKleyman) [inline docs/metrics.asciidoc]: `https://www.elastic.co/guide/en/apm/agent/java` should probably be `https://www.elastic.co/guide/en/apm/agent/dotnet`, right? :)

File: docs/metrics.asciidoc [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+endif::[]
+
+[[metrics]]
+== Metrics
+
+The .NET agent tracks certain system and application metrics.
+Some of them have built-in visualizations and some can only be visualized with custom Kibana dashboards.
+
+These metrics will be sent regularly to the APM Server and from there to Elasticsearch.
+You can adjust the interval with the setting <<config-metrics-interval>>.
```
Comment (SergeyKleyman) [inline docs/metrics.asciidoc]: `config-metrics-interval` link is not in this doc but in `configuration.asciidoc`

File: docs/metrics.asciidoc [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+*`system.memory.actual.free`*::
++
+--
+type: long
+
+format: bytes
+
+Platform: Windows and Linux only.
+
+Actual free memory in bytes.
```
Comment (SergeyKleyman) [inline docs/metrics.asciidoc]: There is already `format: bytes` above and this is the only memory related metric description that reiterates that the units are bytes. I would just leave `Actual free memory`  for consistency.

**Pull request merged**